From a57a3fbb1a22544ce96e4dbe423694e982d54700 Mon Sep 17 00:00:00 2001
From: Jes Sorensen <Jes.Sorensen@redhat.com>
Date: Wed, 20 Jul 2016 16:13:06 -0400
Subject: [PATCH] rtl8xxxu: Enable TX report timer on 8188eu

The 8188eu uses the same TX report timer as found on the 8723b.

Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
---
 drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c
+++ b/drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c
@@ -522,6 +522,7 @@ struct rtl8xxxu_fileops rtl8188eu_fops =
 	.init_phy_rf = rtl8188eu_init_phy_rf,
 	.usb_quirks = rtl8188e_usb_quirks,
 	.writeN_block_size = 128,
+	.has_tx_report = 1,
 	/*
 	 * Use 9K for 8188e normal chip
 	 * Max RX buffer = 10K - max(TxReportSize(64*8), WOLPattern(16*24))
